android/app/src/main/java/app/organicmaps/sdk/widget/placepage/PlacePageButtonFactory.java

```diff
@@ -0,0 +1,6 @@
+package app.organicmaps.sdk.widget.placepage;
+
+public class PlacePageButtonFactory
+{
+  public native static boolean nativeHasRecentlyDeletedBookmark();
```

Comment:
What is the benefit of defining it here? Less files => easier to support and navigate.